src/QsCompiler/Tests.Compiler/Tests.Compiler.fsproj

```diff
@@ -17,6 +17,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Compile Include="SandboxInputSchemaTests.fs" />

```

Comment:
Once you rename the Bond-related files, it would be good to rename the test file too.